src/psyclone/psyir/nodes/call.py

```diff
@@ -750,70 +766,63 @@ def _get_argument_routine_match(self, routine: Routine):
             # TODO #759: Optional keyword is not yet supported in psyir.
             # Hence, we use a simple string match.
             if ", OPTIONAL" not in str(routine_arg.datatype):
+                routine_arg_name = routine_arg.name.replace("\n", "")
```

Comment:
Are the first two `replace`s actually necessary? I would assume that the name attribute is always a string without newlines.